### ISSUE TITLE
Specifying level causes errors

### ISSUE THREAD

**Issue by JakeGinnivan** [fcomb/redux-logger#135]
Unsure what is happening here or why but when I specify the level things go strange.

Given this setup
```
const logger = createLogger({
  colors: false,
  level: {
    prevState: false,
    action: 'info',
    nextState: false,
    error: false
  }
})
```

I am getting `undefined is not a constructor (evaluating 'logger[level]("action", formattedAction)')` in karma.

When debugging the logger is set to `Console` which has a single property `MemoryInfo`. Have you seen anything like this? Using karma with webpack

**Comment by behl1** [comment]
Getting the same issue. +1

**Comment by behl1** [comment]
@JakeGinnivan Did you solve this already?

**Comment by theaqua** [comment]
Fixed in [2.6.1](https://github.com/fcomb/redux-logger/releases/tag/2.6.1).
Sorry for the delay, I was in another city on holidays.
